versioned_docs/version-v2.13.x/user-guide/cli-using-usingcli.md

```diff
@@ -4,7 +4,11 @@ In this section, learn about how to use Zowe CLI, including connecting to the ma
 
```

Comment:
"In this section, learn how to use Zowe CLI ..." maintains consistency with how you describe it elsewhere